src/QirRuntime/public/ResourceStatistics.hpp

```diff
@@ -0,0 +1,55 @@
+#pragma once
+#include <sstream>
+
+namespace Microsoft
+{
+namespace Quantum
+{

```

Comment:
Shouldn't everything in this file be part of the resource estimator implementation, rather than part of the public API?